### DISCUSSION TITLE
Custom Themes

### DISCUSSION THREAD

**Discussion by vHanda** [GitJournal/GitJournal#816] [Feature Requests]
We could potentially even copy popular themes from vscode.

---
> Please vote on this issue with a 👍reaction. It's one of the primary mechanisms used to prioritize issues.

**Comment by madeindjs** [comment]
I would love to see a true black theme (AMOLED). You can take a look at Putch Black theme for VSCode.

**Comment by Iglum** [comment]
This Flutter Material Theme editor looks nice.
Edit: Most of the time its not working at all.

[🎨 Panache](https://github.com/rxlabz/panache)


How would we applay **theme.dart file** to GitJournal in an efficient way? Cheers

Would it not be nice to have the Dark Theme switcher moved to similar place as telegram app? or the button can be the GitJournal logo like a secret 😍
![ezgif com-resize](https://user-images.githubusercontent.com/33994566/89734013-a3024180-da59-11ea-8a7e-d6713ebe4cd6.gif)

**Comment by vHanda** [comment]
@Iglum : https://twitter.com/GitJournal_io/status/1301678153912537088 :)

Telegram's animation is way way better - I'm not sure how to implement it.
